src/systemcmds/top/CMakeLists.txt

```diff
@@ -30,13 +30,14 @@
 # POSSIBILITY OF SUCH DAMAGE.
 #
 ############################################################################
+
 px4_add_module(
 	MODULE systemcmds__top
 	MAIN top
-	PRIORITY "SCHED_PRIORITY_MAX"
-	COMPILE_FLAGS
+	PRIORITY "SCHED_PRIORITY_MAX - 7"
```

Comment:
Maybe a comment why it's -7 would make sense?

Reply:
Comment added.